README.md

```diff
@@ -43,12 +43,22 @@ This project is formatting with [black](https://github.com/psf/black). To autofo
 can use the black command line tool:
 
 ```
-$ black src tests --config setup.cfg
+$ black src tests --config pyproject.toml
```

Comment:
[nit] If this is being run the root directory, I think black will automatically detect and use the pyproject.toml file, i.e. I don't think you need to specify `--config`.

Reply:
Makes sense - I've updated this accordingly